ooasp/encodings/ooasp_simple.lp

```diff
@@ -18,13 +18,8 @@ check_permanent_cv. % Checks for permanent constraints apply
 #include "ooasp_check.lp". % Check section
 
 #include "ooasp_symmetry.lp". % Symmetry breaking constraints
-#include "ooasp_upper_bound.lp". % Calculates global needs
+#include "ooasp_smart_generation.lp". % Functions for smart generation
 
-#show lb_at_least/6.
-#show upper_filled/5.
-#show lower_global/5.
+#show .
 
-#program domain(new_object, cls).
-
-:- user(ooasp_isa(C,new_object  )),
```

Comment:
@susuhahnml Do you know why? Was this only temporary or are we not using the `user(...)` predicates anymore?

Reply:
Yes, at the moment we are adding the user predicate as atoms, not assumptions so this was not really making any difference since it always the case bacause of how we add things. We will add this properly when we include interactivity.